### ISSUE TITLE
RelatedFieldAjaxListFilter not working

### ISSUE THREAD

**Issue by a1Gupta** [geex-arts/django-jet#134]
I cloned the django-jet-demo and tried to use RelatedFieldAjaxListFilter as described [here](https://github.com/geex-arts/django-jet/blob/master/docs/autocomplete.rst).

I modified the PersonAdmin list_filter as shown below-

```
    list_filter = (
        ('address', RelatedFieldAjaxListFilter),
        'company',
    )
```

But after this no related fields are visible in dropdown. Screenshots are attached.

**Without RelatedFieldAjaxListFilter:**
![screenshot from 2016-10-20 18 53 00](https://cloud.githubusercontent.com/assets/5560733/19561637/25cd23c8-96f7-11e6-9bae-fd44575259d3.png)

**With RelatedFieldAjaxListFilter:**
![screenshot from 2016-10-20 18 50 15](https://cloud.githubusercontent.com/assets/5560733/19561636/25c3c530-96f7-11e6-87fd-e01d2e303c33.png)

**Comment by andersondborges** [comment]
I am having the same problem.

**Comment by SalahAdDin** [comment]
Do you use some packages that change the select input box performance?

**Comment by andersondborges** [comment]
I think the problem is the RelatedFieldAjaxListFilter do not set ajax class in the filter and the toolbar.js do not add ajax class on the Select too.

**Comment by SalahAdDin** [comment]
Did you try solve it adding the ajax class?

**Comment by andersondborges** [comment]
I rewrote the filter.html set everything in select html adding the ajax_attrs from the class and changelist-filter-select class to apply the filter event.

**Comment by SalahAdDin** [comment]
Please make a PR :D

**Comment by f1nality** [comment]
@a1Gupta thanks for report. Indeed this bug appeared in 0.9+ versions. Fixed in dev branch and will be released in 1.0.3
